src/client/components/logo/logo.tsx

```diff
@@ -0,0 +1,28 @@
+import type { Location, History } from 'history';
+import PropTypes from 'prop-types';
+import React, { Validator } from 'react';
+import { match, RouteComponentProps, withRouter } from 'react-router-dom';
+
+class Logo extends React.Component<RouteComponentProps> {
+  static get propTypes() {
+    return {
+      match: PropTypes.object.isRequired as Validator<match>,
+      location: PropTypes.object.isRequired as Validator<Location<unknown>>,
+      history: PropTypes.object.isRequired as Validator<History<unknown>>,
+    };
+  }
+
+  render() {
+    const { history } = this.props;
```

Comment:
We could, I suppose, but they are part of the route props, so I'm not sure if it makes sense. That said, when using typescript, the value of `propTypes` decreases a lot anyways, because the static type checking typically is more detailed anyways than what you do at runtime with `propTypes`. I wanted to discuss at some point how we want to deal with that in general.

Reply:
Yes, I also thought the same during the review. I think we should aim for removing the `propTypes` once everything is migrated to TypeScript.